ucfcs/GrowthPlus
# Julio/realm listeners

Merge after julio/childScreenProgress PR. 
This PR implements a realm listener for create child account and refactors back button functionally in landing page, parent portal, child portal and going back to child portal from roadmap.

To test:
- Do a fresh install of the app.
- Create a parent and test navigation of the back buttons.
- Start at: Landing page -> select avatar -> create child (DON'T CREATE A CHILD YET), use back buttons, it should take you back to landing page. 
- Start at: Parent portal -> follow same progression as previous bullet, use back buttons, it should take you back to parent portal.   
- Ensure back buttons are working properly then test creating children, create six children, make sure in each instance it will take you back to landing page. 
- Bug where it is stuck on screen shouldn't be there anymore and the buttons are disable after clicking create.
- You might need to wait a little bit before the app takes you back to landing page.
